### DISCUSSION TITLE
Simplify building Teams meetings apps

### DISCUSSION THREAD

**Discussion by waldekmastykarz** [pnp/generator-teams#170] [Ideas]
When building Teams meetings apps, there are some specific implementation steps to get information about the meeting, participants, their roles, etc. It would be helpful if the project scaffolded by yo teams offered an easy way to get this information.

**Comment by stephanbisser** [comment]
Definitely! I already added an issue for this here https://github.com/pnp/generator-teams/issues/235 feel free to add your comments on this!
